lib/search/service.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'search/response'
+
+module Search
+  class Service < Common::Client::Base
```

Comment:
Can you pls add in some Yard docs explaining that this is wrapper for the Search.gov API, with links out to their API docs.